frontend/projects/marketplace/src/pages/show/additional/additional.component.html

```diff
@@ -1,9 +1,16 @@
 <ion-item-divider>Additional Info</ion-item-divider>
-<ion-card>
+<ion-card *ngIf="pkg.manifest as manifest">
```

Comment:
since `pkg` is asserted to exist (`@Input() pkg!: MarketplacePkg`) do we actually need the if condition here? Is it just for cosmetic or loading reasons?

Reply:
Correct, I do this just so I don't have to do pkg.manifest over and over

Reply:
Is this best practice? Or would having another variable in the component that is just for the manifest be better?

Reply:
This is perfectly fine